custom-routes-vis/src/main/webapp/index.html

```diff
@@ -15,20 +15,39 @@
     <div class="row">
       <div class="col-sm-10 offset-sm-1">
         <h1>Custom Routes Visualization</h1>
+        <div class="modal fade" id="custom-coordinates-warning-modal" tabindex="-1" role="dialog" aria-hidden="true">
+          <div class="modal-dialog">
+            <div class="modal-content">
+              <div class="modal-header">
+                <span class="modal-title" id="custom-coordinates-warning-text"></span>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
         <div id="map"></div>
         <br>
       </div>
     </div>
     <div class="row">
       <div class="col-sm-2 offset-sm-1">
         <h5>Origin Coordinates:</h5>
+        <input type="number" id="origin-lat" placeholder="Enter latitude">
+        <input type="number" id="origin-lng" placeholder="Enter longitude">
+        <button type="button" class="btn btn-success" onclick="showCustomCoordinatesMarker('origin')" id="show-origin-custom-marker">Submit Custom Coordinates</button>
```

Comment:
If we are making this have a class of `btn-success` maybe we should change the place marker button to be `btn-primary` or some other color.